pitest/src/main/java/org/pitest/mutationtest/engine/gregor/GregorMutater.java

```diff
@@ -118,8 +117,12 @@ public Mutant getMutation(final MutationIdentifier id) {
   }
 
   private Predicate<MethodInfo> filterMethods() {
-    return and(this.filter, filterSyntheticMethods(),
-        isGeneratedEnumMethod().negate(), isGroovyClass().negate());
+    return Stream.of(
```

Comment:
Hmm stream or method chain. Seems to be a matter of taste?

Reply:
I changed it to method chain.